src/modules/storage.js

```diff
@@ -57,3 +65,55 @@ window.addEventListener('DOMContentLoaded', () => {
   }
   addList();
 });
+
+// input field of list clicked event
+
+/*
+listsList.addEventListener('focusin', (e) => {
+  if(e.target.className.includes('single-list-input')){
+    const parent = e.target.parentNode.parentNode;
+    parent.querySelector('.single-list-action-button').innerHTML =
+    '<button class = "delete-btn" onclick="removeList('${list}')"> // eslint-disable-line
+    <i class="fa-solid fa-trash-can"></i></button>';
+  }
+})
+listsList.addEventListener('focusout', (e) => {
+  if(e.target.className.includes('single-list-input')){
+    const parent = e.target.parentNode.parentNode;
+    parent.querySelector('.single-list-action-button').innerHTML = '<button class = "move-btn">
+    <i class="fa-solid fa-ellipsis-vertical"></i></button>';
+
+    const inputValue = e.target.value
+    const {id} = parent.id;
+
+    listArray.forEach(list => {
+      if(id === list.index){
+        list.description = inputValue
+      }
+    })
+
+    localStorage.setItem('listData', JSON.stringify(listArray));
+  }
+})
+*/
+// checkbox event
+
+listsList.addEventListener('change', function (e) { // eslint-disable-line
+  if (e.target.className.includes('checkbox')) {
+    const { checked } = e.target;
+    const text = e.target.parentNode.querySelector('.single-list-input');
+
+    if (checked) {
+      text.classList.add('line-through');
+    } else {
+      text.classList.remove('line-through');
+    }
+  }
+});
```

Comment:
- [ ] When I checked the task, the status is not changing in the local storage. Please fix it.